int64 throttle
---
int64 code